core/src/main/java/com/linecorp/armeria/common/Cookie.java

```diff
@@ -58,21 +58,49 @@
      *
      * @param name the name of the {@link Cookie}
      * @param value the value of the {@link Cookie}
+     *
+     * @deprecated Use {@link #ofSecure(String, String)} instead to create a secure {@link Cookie}.
      */
+    @Deprecated
```

Comment:
Found a couple test cases where this method is still being used.

- HttpHeaderCachedValuesTest
- CookieSuite

Is it possible to update these to use `Cookie#ofSecure` as well?

Reply:
I also found some instances of `Cookie#of` in the javadoc of our public API
Is it also possible to update occurrences here as well?

- AbstractHttpRequestBuilder

Reply:
Oh Thanks. Let me check it again

Reply:
Keep using `Cookie#ofSecure` in HttpHeaderCachedValuesTest due to https://github.com/line/armeria/issues/3968